### PR TITLE
feats: users can customize the address (#4)

### PR DESCRIPTION
**Pull request by Joker2770**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -47,9 +43,9 @@ where
     I2C::Error: Into<Error<I2C::Error>>,
 {
     /// Creates a new driver from an I2C peripheral.
-    pub fn new(i2c: I2C) -> Self {
```
Comment (MrBuddyCasino) [inline src/lib.rs:50]: Maybe you can preserve this with the old default value in order not to break the API? 
Depending on this, the crate's version should be incremented (either major or minor).

Comment (Joker2770) [reply src/lib.rs:50]:
```rust
    /// Creates a new driver from an I2C peripheral.
    pub fn new(i2c: I2C) -> Self {
        MCP9808 {
            addr: SlaveAddress::Default.into(),
            i2c,
        }
    }

    /// Change i2c address
    pub fn set_address(&mut self, addr: SlaveAddress) -> u8 {
        self.addr = addr.into();
        self.addr
    }
```
How about this?

Comment (MrBuddyCasino) [reply src/lib.rs:50]: Sorry been traveling again, looks good! If you increment the crate version I can merge it.

**Comment by MrBuddyCasino** [comment]
Thx for your PR! Sorry it took so long.

**Pull request merged**